README.md

```diff
@@ -30,6 +30,7 @@ type = :customer_data_request
 SegmentAPI.track(
   SegmentAPI.event(app_slug, type),
   shopify_domain,
-  %{data: %{foo: "bar"}}
+  %{data: %{foo: "bar"}},
+  {integrations: %{All: true, Salesforce: false}}
```

Comment:
missing a %
should be `%{integra...`

Reply:
https://frinkiac.com/video/S05E05/SSv5ncW99B11fAdvxYrRH2FV_KQ=.gif